locales/en-US/app.ftl

```diff
@@ -30,9 +30,19 @@ Home--load-from-url-submit-button =
     .value = Load
 
 Home--documentation-button = Documentation
+Home--menu-button = Enable Profiler Menu Button
+Home--menu-button-instructions =
+    Enable the profiler menu button to start recording a performance
+    profile in Firefox, then analyze it and share it with profiler.firefox.com.
+
+Home--addon-button = Install add-on
+Home--addon-button-instructions =
+    Install the Gecko Profiler Add-on to start recording a performance
```

Comment:
Is there still an actual add-on with this name? The only thing I can find is a deprecated add-on
https://github.com/firefox-devtools/Gecko-Profiler-Addon

Reply:
Yeah, we still support this for the ESR versions (I don't remember if this is only for 70 or for both 70 and 78).

Reply:
Should we treat `Gecko Profiler Add-on` as a brand?

Reply:
Possibly, but I'm leaving this to Julien (70 is already unsupported, and 78 won't last much longer, so this might be destined for removal soon).

Reply:
Yeah, any direction works for me too.